peer.js

```diff
@@ -68,6 +68,9 @@ function TChannelPeer(channel, hostPort, options) {
     this.boundOnPendingChange = onPendingChange;
     this.scoreRange = null;
 
+    this.nextConnAttemptTime = 0;
+    this.nextConnAttemptDelay = 0;
```

Comment:
Should these values be reset when a connection is successfully identified? Peers are long lived. If these values only grow, won’t connecting to that peer become permanently sluggish?